src/liberty/libertyProject.ts

```diff
@@ -213,6 +213,10 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 
 	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	public async getChildren(element?: LibertyProject): Promise<LibertyProject[]> {
+		
+		if (this.projects.size <= 0) {
+			return Promise.resolve([]);
+		}
```

Comment:
what is this code for?

Reply:
initially i thought we need to add this check in getChildren() in order to get the welcome view. But now i have tested and found that this check is not necessary for the welcome view. Whenever tree view is empty, vscode itself is showing the welcome view. So removing this condition.